Nie-ontologies/Generic-ontologies/publishing.ttl

```diff
@@ -358,7 +358,7 @@ publish:hasElectronicPublication
 publish:hasWebPublication
 	rdfs:isDefinedBy <http://e-editiones.ch/ontology/publishing#>;
 	a owl:ObjectProperty;
-	rdfs:label "has web publication"@en, "hat Web-Publikation"@de, "a une publication web"@fr, "ha una pubblicazione web"@it;
+	rdfs:label "has web publication"@en, "hat Onlinepublikation"@de, "a une publication web"@fr, "ha una pubblicazione web"@it;
```

Comment:
Is 'Onlinepublikation' not more general than 'Web-Publikation'?

Reply:
I would say it is the same: [https://de.m.wikipedia.org/wiki/Web-Publikation?title=Web-Publikation&redirect=no](https://de.m.wikipedia.org/wiki/Web-Publikation?title=Web-Publikation&redirect=n)
Normally in German one speaks of Online instead of Web. Nevertheless changed it because of consistency.

Reply:
Commits 716948b and 2da6731